### PR TITLE
completed dom-manipulation exercise  - Colin Macrae

### PR DESCRIPTION
**Pull request by colin-macrae**

### CODE REVIEW

File: dom-manipulation/main.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    // document.getElementsByClassName('.cold');
+    if ($timesClicked <= 4) {
+      $hotButton.className = 'hot-button cold';
+    } else if ($timesClicked <= 7) {
+      $hotButton.className = 'hot-button cool';
+    } else if ($timesClicked <= 10) {
+      $hotButton.className = 'hot-button tepid';
+    } else if ($timesClicked <= 13) {
+      $hotButton.className = 'hot-button warm';
+    } else if ($timesClicked <= 16) {
```
Comment (shawnkost) [inline dom-manipulation/main.js:19]: The conditionals should actually just be less than, rather than less than or equal to.

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change Colin!

File: dom-manipulation/main.js (hunk 1 above)
Comment (colin-macrae) [reply dom-manipulation/main.js:19]: Right.  I do see that now in the assignment instructions.  Thx.

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**